docs/source/getting-started.rst

```diff
@@ -0,0 +1,90 @@
+
+===============
+Getting started
+===============
+
+This getting started guide walks you through using Anaconda Project for the first time. 
+
+**After completing this guide, you will be able to:**
+
+* Create a project containing a Bokeh app
+* Package and share the project
+* Run the project with a single command
+
+This guide is for all platforms: Windows, OS X and Linux.
+
+If you have not yet installed and started Anaconda Project, follow the :doc:`Install instructions <install>`.
+
+NOTE: Windows users using Python 3.5.1 should upgrade to Python 3.5.2+ with the command ``conda update python`` to avoid an issue with Windows and Python 3.5.1.
+
+
+Create a project containing a Bokeh app
+=======================================
+
+We'll create a project directory called ``clustering_app``. At the command prompt, switch to a directory ``clustering_app`` and initialize the project::
+
+    $ mkdir clustering_app
+    $ cd clustering_app
+    $ anaconda-project init
+    Project configuration is in /User/Anaconda/My Anaconda Projects/clustering_app/anaconda-project.yml
+
+Inside your ``clustering_app`` project directory, create and save a new file named ``main.py`` with the `Bokeh clustering example <https://raw.githubusercontent.com/bokeh/bokeh/master/examples/app/clustering/main.py>`_. Learn more about `Bokeh <http://bokeh.pydata.org/en/latest/>`_.)
+
+We need to add the packages that the Bokeh clustering demo depends on: Bokeh, pandas, scikit-learn and numpy. Open the ``anaconda-project.yml`` file and edit the packages section with:
+
+.. code-block:: yaml
+
+    packages:
+      - python=3.5
+      - bokeh=0.12.4
+      - numpy=1.12.0
+      - scikit-learn=0.18.1
```

Comment:
A somewhat odd thing that happens here is that `anaconda-project init` already put `anaconda` in the `env_specs: default: packages:`; the default env spec will then contain all of the above plus anaconda. Maybe we should leave the docs as-is here, and I will open an issue so that by default I'll put `anaconda` up here in the global packages section. I wouldn't want to put "and also delete the random anaconda in the env_specs section" here in the docs, and for now it isn't really hurting anything, just bloating the env.

Unfortunately ruamel_yaml doesn't format things in the above way by default either, it does this ugly `[anaconda]` thing all on one line... 

Anyway suggestion is no change here and I'll go clean up what `init` does